### ISSUE TITLE
Bug: Non-responsive program timeline on small screen sizes

### ISSUE THREAD

**Issue by Rahulm2310** [anitab-org/anitab-org.github.io#223]
### Describe the bug
The program timeline under the Programs tab is not responsive on small screen sizes.

### To Reproduce
1. Go to https://anitab-org.github.io/
2. Click on Programs
3. Shrink down your screen size
4. See overlapping timeline

### Expected behavior
Program timeline should be responsive on all screen sizes.

### Screenshots
![program-timeline-non-resp](https://user-images.githubusercontent.com/54268438/107077219-8ed5ac00-6812-11eb-87b7-706514898c01.JPG)

### Desktop (please complete the following information):
 - OS: Windows 10
 - Browser: Chrome
 - Version : 84

**Comment by eternal-dreamer** [comment]
Can I work on this issue?

**Comment by Rahulm2310** [comment]
@annabauza If you think it is not a first timer issue then I would like to take this up. Thanks 😊

**Comment by annabauza** [comment]
Yes. Just a reminder we migrating to react js. So maybe we should include it in the issue. Thanks.

**Comment by Rahulm2310** [comment]
>So maybe we should include it in the issue
@annabauza Can you please elaborate  ?

**Comment by annabauza** [comment]
We currently using react native and desition was made to migrate to react js. @Shruti3004 is migrating first navigation and then home page

**Comment by Rahulm2310** [comment]
@annabauza  So, you mean this will be covered in the migration process?

**Comment by annabauza** [comment]
You should include migration as requirement.

**Comment by shades-7** [comment]
@Rahulm2310 may i work on this?

**Comment by RiShI11-0902** [comment]
Hey can I work on this, I am React JS developer and I Will try my best to fix this.
